.github/workflows/merge-to-stage.js

```diff
@@ -0,0 +1,233 @@
+const {
+  slackNotification,
+  getLocalConfigs,
+  pulls: { addLabels, addFiles, getChecks, getReviews },
+} = require('./helpers.js');
+
+// Run from the root of the project for local testing: node --env-file=.env .github/workflows/merge-to-stage.js
+const prTitle = '[Release] Stage to Main';
+const seen = {};
+const requiredApprovals = process.env.REQUIRED_APPROVALS || 2;
```

Comment:
We don't require this at the moment either - 2 approvals & that counts. Code owners are just automatically looped in/requested for reviews

Reply:
true, but it's a good point from @overmyheadandbody , maybe in future we will need this..

Reply:
Github offers this OOTB for a whole repo, we can still decide to enable this at some point
![Screenshot 2024-05-02 at 16 38 28](https://github.com/adobecom/milo/assets/39759830/0378b91e-cc87-46b5-aca6-55b2737558ff)

So far it hasn't been an issue and hope it won't be in the future either - but the option is there

Reply:
Hmm, maybe someone de-activated it recently? A couple of months ago at least one code owner had to review the PR, even though it had more than 2 approvals already. And I'd say it's fair enough, that's one of the reasons we have the codeowners in the first place - automatically assign the specialized person and have them give their two cents. This might deserve some further discussion to understand who and why disabled this rule.